spec/models/report_spec.rb

```diff
@@ -0,0 +1,36 @@
+describe Report do
+  it "#client_query" do
+    report = create(:report)
+    expect(report.client_query.to_s).to eq "client_data.amount:<123"
```

Comment:
these values feel a little bit mysterious because they come from the Report factory's default vals. perhaps we could set them explicitly in the factory instantiation so it is clearer what the method has to do with object attrs?